tests/renderering/RoundPixels.test.ts

```diff
@@ -15,23 +16,13 @@ describe('Round Pixels', () =>
 {
     it('Round pixels should set correctly when passed as an option', async () =>
```

Comment:
i think this is a different test. Perhaps we have both?

one is testing the renderer and one is testing the Application